app/scripts/controllers/metametrics.js

```diff
@@ -768,6 +768,8 @@ export default class MetaMetricsController {
       ///: END:ONLY_INCLUDE_IN
       [MetaMetricsUserTrait.SecurityProviders]:
         metamaskState.transactionSecurityCheckEnabled ? ['opensea'] : [],
+      [MetaMetricsUserTrait.SecurityProviders]:
```

Comment:
Just curious, why the key is same as the previous one? Shouldn't the keys be unique?